src/drivers/driver_zephyr.c

```diff
@@ -236,6 +237,422 @@ static void wpa_drv_zep_event_proc_unprot_disassoc(struct zep_drv_if_ctx *if_ctx
 			event);
 }
 
+struct phy_info_arg {
+	u16 *num_modes;
+	struct hostapd_hw_modes *modes;
+	int last_mode, last_chan_idx;
+	int failed;
+	u8 dfs_domain;
+};
+
+static void phy_info_freq_cfg(struct hostapd_hw_modes *mode,
+		struct hostapd_channel_data *chan,
+		struct wpa_supp_event_channel *chnl_info)
+{
+	u8 channel;
```

Comment:
initialize to 0. Also insert a blank line between variable declaration and code.